### DISCUSSION TITLE
Update removed license

### DISCUSSION THREAD

**Discussion by Bezt-of-the-Best** [aclap-dev/video-downloadhelper#831] [Help and Q&A]
### Your question

The update to Video DownloadHelper removed my license that I have had for a long time.  Now I can't use the program any more and the system only recognizes licenses within the last year.  Help.  Please restore as there appears to be no way to do so on line.

### Addon & CoApp version and details

Video DownloadHelper
Version 9.0.2.3 (stable)
Browser locale: en-US
Production build
Built on 2024-05-27
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla

Platform Win x86-64
Browser Mozilla Firefox 115.11.0

**Comment by jcfrog** [comment]
Hi,
You probably just need to revalidate your license key: https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#validate-your-license-key
